app-crypt/acme-sh/acme-sh-2.7.9.ebuild

```diff
@@ -0,0 +1,46 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+MY_PN="${PN/-/.}"
+MY_P="${MY_PN}-${PV}"
+
+DESCRIPTION="An ACME Shell script"
+HOMEPAGE="https://github.com/Neilpang/acme.sh"
+SRC_URI="https://github.com/Neilpang/${MY_PN}/archive/${PV}.tar.gz -> ${MY_P}.tar.gz"
+
+LICENSE="GPL-3"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="libressl"
+
+RDEPEND="net-misc/curl
+	|| ( net-analyzer/netcat6 net-analyzer/openbsd-netcat )
+	|| ( www-servers/apache:2 www-servers/nginx:0 net-misc/socat )
+	virtual/cron
+	libressl? ( dev-libs/libressl:0= )
+	!libressl? ( dev-libs/openssl:0= )"
+
+S="${WORKDIR}/${MY_P}"
+
+src_install() {
+	einstalldocs
+	newdoc dnsapi/README.md README-dnsapi.md
+	newdoc deploy/README.md README-deploy.md
+
+	keepdir /etc/acme-sh
+	doenvd "${FILESDIR}"/99acme-sh
+	insinto /etc/bash/bashrc.d
+	doins "${FILESDIR}"/acme.sh
+
+	insinto /usr/share/acme.sh
+	doins acme.sh
+	insinto /usr/share/acme.sh/dnsapi
+	doins -r dnsapi/*.sh
+	insinto /usr/share/acme.sh/deploy
+	doins -r deploy/*.sh
+
+	dosym ../share/acme.sh/acme.sh usr/bin/acme.sh
+	fperms +x /usr/bin/acme.sh
```

Comment:
Well, given this affects `acme.sh` installed above, why not just install it with `exeinto`/`doexe`?

Reply:
Done.